apollographql/odyssey-lift-off-lab
# Package updates

- ApolloServer 3
- npm audit fixes
- react-markdown upgrade

**To test starting point:**
Run `npm install && npm start` in the following folders:
- `server`
- `client`

App should be running with no errors, see homepage, navigate to track detail page.

**To test final stage:**
Run `npm install && npm start` in the following folders:
- `final/server`
- `final/client`

App should be running with no errors, see homepage, navigate to track detail page and navigate to module detail page.
